compose/logging/logging.yaml

```diff
@@ -1,31 +1,67 @@
 version: '3.7'
 
 services:
-
   elasticsearch:
-   image: docker.elastic.co/elasticsearch/elasticsearch:6.2.4
-   container_name: elasticsearch
-   ports:
-    - "9200:9200"
-   volumes:
-    - elasticsearch-data:/usr/share/elasticsearch/data
-   networks:
-    - portfolio-manager
+    build:
+      context: elasticsearch/
+      args:
+        ELK_VERSION: $ELK_VERSION
+    volumes:
+      - type: bind
+        source: ./elasticsearch/config/elasticsearch.yml
+        target: /usr/share/elasticsearch/config/elasticsearch.yml
+        read_only: true
+      - type: volume
+        source: elasticsearch
+        target: /usr/share/elasticsearch/data
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+    environment:
+      ES_JAVA_OPTS: "-Xmx256m -Xms256m"
```

Comment:
curious what that does

Reply:
https://github.com/deviantony/docker-elk/blob/4156c0cb9978f336c3a8f155f8b61e447d632584/README.md#jvm-tuning